### PR TITLE
React/Redux Group Project

### PR DESCRIPTION
**Pull request by SimbaWANGU**
In this Pull Request, the following changes have been introduced:
1. Basic configuration and setup of the app
2. Fetched data from the provided API and updated the Redux store
3. Rendered UI lists with data from the Redux Store
4. Wrote actions and reducers for booking rockets and dragons and joining missions
5. WWrote actions and reducers for cancelling rockets and dragons and leaving missions
6. Used conditional rendering to render UI components on booking a rocket/dragon and joining a mission
7. Rendered my Profile section based on rocket/dragon/mission status

### CODE REVIEW

File: src/__tests__/DragonsContainer.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import DragonContainer from '../components/DragonsContainer';
+
+it('matches Dragon container', () => {
+  const tree = renderer.create(
+    <DragonContainer />,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
```
Comment (Rustamxon7) [inline src/__tests__/DragonsContainer.test.js:9]: - [ ] Your project is working fine! But you have some failing tests as you can see on this image:


Screen Shot 2022-06-17 at 4 01 49 PM

Please make sure, all tests passing to make your project approvable!

File: src/__tests__/RocketContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    rocket_name: 'Rocket',
+    description: 'Lorem ipsum dolor sit amet',
+    flickr_image: 'https://img.example.com',
+    reserved: false,
+  };
+  it('Page renders as expected', () => {
+    const component = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <RocketContainer
```
Comment (Rustamxon7) [inline src/__tests__/RocketContainer.test.js:21]: - [ ] Please make sure, all tests passing to make your project approvable!


Screen Shot 2022-06-17 at 4 01 31 PM

File: README.md [hunk 3]
```diff
@@ -1 +1,68 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone who will take a glance to this project
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Rustamxon7) [inline README.md:68]: - [ ] [OPTIONAL] Kindly add MIT License to your project.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimbaWANGU** [comment]
@Rustamxon7 Thank you for the review 😄 
We will act on the requested changes ASAP.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @team 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**